### PR TITLE
Update fix calculations cmm

### PR DESCRIPTION
**Pull request by lomamech**

### CODE REVIEW

File: client/src/modules/stock/inventories/templates/cmm.cell.html [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <div class="ui-grid-cell-contents">
   <span class="text-right">
-    {{row.entity.cmm.algo_msh}}
+    {{row.entity.avg_consumption}}
```
Comment (jniles) [inline client/src/modules/stock/inventories/templates/cmm.cell.html:3]: :+1:

File: client/src/modules/stock/settings/stock-settings.html [hunk 2]
```diff
@@ -97,7 +97,7 @@
           <!-- average_consumption_algo -->
           <div id="average_consumption_algo" class="form-group"
             ng-class="{'has-error' : StockSettingsForm.$submitted && StockSettingsForm.average_consumption_algo.$invalid }">
-            <label class="control-label" translate>FORM.LABELS.ALGO_MONTHLY_CONSUMPTION</label> DAF PINK
+            <label class="control-label" translate>FORM.LABELS.ALGO_MONTHLY_CONSUMPTION</label>
```
Comment (jniles) [inline client/src/modules/stock/settings/stock-settings.html:100]: 😆

**Comment by lomamech** [comment]
Allo @mbayopanda  just to let you know that all the tests pass and if you could rewrite all the tests related to the stock as agreed while making sure that the data as well as the calculations are correct

**Review by jniles — APPROVED** [review]
I'm going to go ahead and merge this so we can all write tests based off of it.

bors r+

**Comment by jniles** [comment]
I'll make subsequent changes in an additional PR :+1:

**Pull request merged**